var/spack/repos/builtin/packages/biobb-common/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class BiobbCommon(PythonPackage):
+    """Biobb_common is the base package required to use the biobb packages"""
+
+    homepage = "https://github.com/bioexcel/biobb_common"
+    git = "https://github.com/bioexcel/biobb_common.git"
+
+    maintainers("d-beltran")
+
+    # Versions
+    version("4.0.0", branch="master")
+
+    # Dependencies
+    depends_on("py-setuptools")
+    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-pyyaml", type=("build", "run"))
+    depends_on("py-requests", type=("build", "run"))
+    depends_on("py-biopython@1.78:1.80", type=("build", "run"))
+
+    # Test
```

Comment:
You can remove this, it's already automatically handled by PythonPackage

Reply:
done :)